moleculemaker/mmli-backend
# fix: errors when running ChemScraper after k8s refactoring

## Problem
After merging the k8s job runner, ChemScraper has trouble locating its output files

## Approach
* fix: bad paths when attempting to fetch and process ChemScraper result files

## How to Test
Test this with https://github.com/moleculemaker/chemscraper-frontend/pull/102

Currently deployed on prod: https://chemscraper.platform.moleculemaker.org

1. Navigate to https://chemscraper.platform.moleculemaker.org/configuration
2. Submit a real (known-good) PDF for analysis
3. Wait for job to complete
    * You should eventually see the results load and display
